### PR TITLE
Feat: locust

### PR DESCRIPTION
**Pull request by deep-diver**

### CODE REVIEW

File: locust/locustfile.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from locust import HttpUser, task
+
+class HelloWorldUser(HttpUser):
```
Comment (sayakpaul) [inline locust/locustfile.py:3]: Let's have a more meaningful name?

File: locust/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+         --spawn-rate SPAWN_RATE \ 
+         --host HOST_ADDRESS
+
+# without UI & manual config
+# the report will be generated to report.html
+$ locust --headless \ 
+         --users NUM_OF_USERS \
+         --spawn-rate SPAWN_RATE \ 
+         --host HOST_ADDRESS \ 
+         --html report.html
```
Comment (sayakpaul) [inline locust/README.md:29]: Will `report.html` be generated in the `locust` directory itself? If so, let's add a note about it. 

Also, are you launching the test from your local machine?

Comment (sayakpaul) [reply locust/README.md:29]: Also, once the clusters are set up on GKE I think we can share the Locust experiments amongst ourselves. I'd need to know the IPs before that of course.

**Review by sayakpaul — CHANGES_REQUESTED** [review]
Minor changes. Once done feel free to merge.

File: locust/README.md (hunk 2 above)
Comment (deep-diver) [reply locust/README.md:29]: @sayakpaul 

I have launched locust test in GitHub's codebase VM, and yes `report.html` will be generated in the same directory (but you can manually do this from the locust UI)

Comment (sayakpaul) [reply locust/README.md:29]: Did you mean Codespace?

Comment (deep-diver) [reply locust/README.md:29]: ah yes sorry about the typo

**Review by sayakpaul — COMMENTED** [review]
Clarification needed.

**Comment by sayakpaul** [comment]
Closing this since Locust load-testing is handled efficiently now. Check `locust` of this repository.

**Pull request closed without merging**